about.html

```diff
@@ -0,0 +1,324 @@
+                            year's NMC is a different one because unlike every other
+                            NMC it's done in June which is the middle of the year and
+                            it's gonna be fun. So do yourself a big favor to forget sorrow.
+                            Don't forget that it's not meant for Nigerians alone, friends
+                            from overseas and other country are welcome to be happy, so don't
+                            let them miss that special and happy moment of their lives. This is
+                            taking place in Nigeria, get your tickets ready so you don't miss the
+                            front row. Stay tuned.
+
+                            </p>
+                            <p><img src="images/about_icon.png" alt="icon"></p>
+                            <p class="about_paragraph_box">June 24th - 25th, 2023 (Sat - Sun) <br>
+                                See you at the Nigeria music concert, see you live!</p>
+
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-12 ">
+                            <p class="about_buttom_paragraph">For any enquiry about NMC 2023 please contact us via
+                                Email! <br>
+                                <span class="organisation"><a href="#">nmc2023@ng.org</a> </span>
+                            </p>
+                        </div>
+                    </div>
+
+
+                </div>
+            </div>
+        </div>
+        <div>
+            <!-- ---------------------Middle section about page  -->
+            <div class="container-fluid about_middle_section">
+                <div class="row">
+                    <div class="col-4"></div>
+                    <div class="col-8">
+                        <div class="about_middle_container">
+                            <h2>Nigeria Music Concert NMC 2023 Logo</h2>
+                            <div class="row">
+                                <div class="col-2"></div>
+                                <div class="col-10">
+                                    <div class="about_divider"></div>
+                                </div>
+                            </div>
+                        </div>
+
+
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-2"></div>
+                    <div class="col-8 about_paragraph_container">
+                        <p>
+                            The NMC logo was originated from its first letters of its title Nigeria Music Concert
+                            It was styled NMC in January 2023 and now called NMC instead of its original title.
+                            Here is how its styled, though simple but easy to call.
+                        </p>
+
+                    </div>
```

Comment:
- [ ] Good job following the design @fauziyahzeeyah, according to the requirements, this section is supposed to be entered, Please make the changes for a better layout. 

![Screenshot 2023-01-14 162612](https://user-images.githubusercontent.com/69077061/212474169-9bed0a92-8a93-4340-9d2e-7cf027c86507.png)